java-components/cli/src/main/java/io/github/redhatappstudio/jvmbuild/cli/builds/BuildLogsCommand.java

```diff
@@ -39,10 +61,19 @@ public class BuildLogsCommand implements Runnable {
     @CommandLine.Option(names = "-n", description = "The build number", defaultValue = "-1")
     int buildNo;
 
+    @CommandLine.Option(names = "-l", description = "Use legacy retrieval")
+    boolean legacyRetrieval = false;
+
+    @CommandLine.Option(names = "-u", description = "URL for Tekton-Results")
+    String tektonUrl = "https://console.redhat.com/";
```

Comment:
This should not have the trailing slash, you end up with 'https://console.redhat.com/:443' otherwise.